src/coreclr/src/classlibnative/bcltype/system.cpp

```diff
@@ -607,9 +607,22 @@ BOOL QCALLTYPE SystemNative::WinRTSupported()
 
 #endif // FEATURE_COMINTEROP
 
+#if defined(TARGET_X86) || defined(TARGET_AMD64)
 
+#if !defined(_MSC_VER)
+#include <cpuid.h>
```

Comment:
You cannot include system headers outside PAL in CoreCLR.

The easiest way to make this work is probably by calling the existing `getextcpuid` method.